src/main/webapp/app/exercises/quiz/manage/re-evaluate/multiple-choice-question/re-evaluate-multiple-choice-question.component.ts

```diff
@@ -29,12 +29,17 @@ export class ReEvaluateMultipleChoiceQuestionComponent {
     // Create Backup Question for resets
     @Input() backupQuestion: MultipleChoiceQuestion;
 
+    /** Status boolean for collapse status **/
```

Comment:
Does this comment actually add any information?

Reply:
I removed the comment since it is not useful.